app/src/main/java/com/github/polybooks/core/database/interfaces/SaleDatabase.kt

```diff
@@ -40,30 +44,32 @@ interface SaleDatabase {
  * A SaleQuery is a builder for a query to the database that will yield Sales.
  * Most methods return themselves for function chaining.
  * */
-interface SaleQuery : Query<Sale> {
+interface SaleQuery : Query<Sale>, Serializable {
```

Comment:
Pourquoi ?

Reply:
C'est nécessaire pour passer une instance de la classe d'une activité à une autre

Reply:
On en avait parlé il y a genre 3 semaines

Reply:
@DGoedtkindtEPFL https://stackoverflow.com/questions/2736389/how-to-pass-an-object-from-one-activity-to-another-on-android),